solar-system/src/org/gearvrf/solarsystem/SolarViewManager.java

```diff
@@ -131,28 +139,31 @@ public void run() {
         GVRSceneObject marsRotationObject = new GVRSceneObject(gvrContext);
         marsRevolutionObject.addChildObject(marsRotationObject);
 
-        GVRSceneObject marsMeshObject = asyncSceneObject(gvrContext,
-                "mars_1k_color.jpg");
+        GVRSceneObject marsMeshObject = asyncSceneObject(gvrContext, "mars_1k_color.jpg");
         marsMeshObject.getTransform().setScale(0.6f, 0.6f, 0.6f);
         marsRotationObject.addChildObject(marsMeshObject);
+        marsMeshObject.setName("MARS");
+        marsMeshObject.getRenderData().enableLightMap();
 
         counterClockwise(sunRotationObject, 50f);
 
         counterClockwise(mercuryRevolutionObject, 150f);
-        counterClockwise(mercuryRotationObject, 100f);
+        // counterClockwise(mercuryRotationObject, 100f);
 
         counterClockwise(venusRevolutionObject, 400f);
-        clockwise(venusRotationObject, 400f);
+        // clockwise(venusRotationObject, 400f);
 
         counterClockwise(earthRevolutionObject, 600f);
-        counterClockwise(earthRotationObject, 1.5f);
+        // counterClockwise(earthRotationObject, 15f);
 
         counterClockwise(moonRevolutionObject, 60f);
 
         clockwise(mMainScene.getMainCameraRig().getTransform(), 60f);
 
         counterClockwise(marsRevolutionObject, 1200f);
-        counterClockwise(marsRotationObject, 200f);
+        // counterClockwise(marsRotationObject, 200f);
+
+        new LightMapManager(gvrContext, R.drawable.lightmap_texture, R.raw.lightmap_info).applyLightMap(gvrContext, mMainScene);
```

Comment:
Does it make sure LightMapManager doesn't get garbage collected? If not, please assign it to a data field and suppress the "unused" warning... Just a suggestion. Sorry about these many comments.